Database.py

```diff
@@ -37,7 +38,10 @@ def addEmployee(self, **kwargs) -> Boolean:
         """
         if len(self.search(Employee_ID=kwargs["Emp_ID"])) > 0:
             return False
-        self.employeeList.append(Employee(**kwargs))
+        emp = Employee(**kwargs)
+        if emp.hashed_password == INVALID_STR and emp.Emp_ID != INVALID_STR:
```

Comment:
Is this necessary? I feel like this should happen on employee initialization, which it is, instead of outside of the constructor

Reply:
it was messing with testing a lot by having it there. also, if the database imports from a file that doesn't have passwords(such as an export with adminInfo=FALSE), then every imported account would have their password reset, which is a security vulnerability that we can say that we fixed on the bug tracker.